cmd/lotus-seed/genesis.go

```diff
@@ -9,6 +9,11 @@ import (
 	"strconv"
 	"strings"
 
+	"github.com/filecoin-project/lotus/chain/vm"
+	"github.com/filecoin-project/lotus/extern/sector-storage/ffiwrapper"
+	"github.com/filecoin-project/lotus/journal"
+	"github.com/filecoin-project/lotus/lib/blockstore"
+	"github.com/filecoin-project/lotus/node/modules/testing"
```

Comment:
This is the same module used in the lotus daemon CLI currently. Maybe this should be moved away from a "testing" module.